### PR TITLE
#3030 expose the init heap and max heap

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3030)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-it/src/test/scala/com/island/ohara/it/WithRemoteWorkers.scala [hunk 1]
```diff
@@ -26,6 +26,12 @@ import org.junit.Before
 import scala.concurrent.ExecutionContext.Implicits.global
 
 abstract class WithRemoteWorkers extends WithRemoteConfigurator {
+  private[this] val zkInitHeap       = sys.env.get("zk.xms").map(_.toInt).getOrElse(1024)
+  private[this] val zkMaxHeap        = sys.env.get("zk.xmx").map(_.toInt).getOrElse(1024)
+  private[this] val bkInitHeap       = sys.env.get("bk.xms").map(_.toInt).getOrElse(1024)
+  private[this] val bkMaxHeap        = sys.env.get("bk.xmx").map(_.toInt).getOrElse(1024)
+  private[this] val wkInitHeap       = sys.env.get("wk.xms").map(_.toInt).getOrElse(1024)
+  private[this] val wkMaxHeap        = sys.env.get("wk.xmx").map(_.toInt).getOrElse(1024)
```
Comment (jackyoh) [inline ohara-it/src/test/scala/com/island/ohara/it/WithRemoteWorkers.scala]: 這些參數需要加到 QA 上嗎？

Comment (chia7712) [reply ohara-it/src/test/scala/com/island/ohara/it/WithRemoteWorkers.scala]: > 這些參數需要加到 QA 上嗎？

感謝提醒，我跑完測試會加上去

**Comment by chia7712** [comment]
retry connector

**Comment by chia7712** [comment]
retry connector

**Comment by jackyoh** [comment]
retry

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
unrelated error

**Pull request merged**